api/src/test/java/org/pmiops/workbench/workspaces/WorkspacesControllerTest.java

```diff
@@ -65,8 +71,8 @@
 import org.pmiops.workbench.cohorts.CohortMaterializationService;
 import org.pmiops.workbench.config.WorkbenchConfig;
 import org.pmiops.workbench.db.dao.CdrVersionDao;
-import org.pmiops.workbench.db.dao.CohortCloningService;
-import org.pmiops.workbench.db.dao.ConceptSetService;
+import org.pmiops.workbench.cohorts.CohortCloningService;
```

Comment:
nit, and probably not fixable here: why is cohorts plural and conceptset singular?

Reply:
We have a lot of naming issues to address 🙃